src/CartItemList.tsx

```diff
@@ -50,6 +50,21 @@ export const CartItemList: React.FC<CartItemListParams> = (props) => {
     handlePage(page)
   };
 
+  const handleRemoveAllItems = () => {
+    setShowLoader(true);
+    removeAllCartItems(mcart)
+      .then(() => {
+        setShowLoader(false);
```

Comment:
You may need to also hide the loader & modal in your catch block in case there's an error thrown.